skills/issue-context/SKILL.md

```
@@ -42,39 +42,39 @@ These branch patterns produce no issue context — files use flat root placement
 
 ## File Placement
 
-Foundation skills (scratchpad, question, commit-msg) use this to determine the target directory:
+Foundation skills (scratchpad, question, commit-msg) use this to determine the target directory. All working files live under `.claude-work/`, organized by type subdirectory.
 
 ### When issue context is detected
 
-Files go in an issue-scoped subdirectory:
+Files go in a type subdirectory within the issue directory:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)